test/distributed/_tensor/test_view_ops.py

```diff
@@ -495,14 +494,14 @@ def test_complex_view_ops(self):
             InputDim(0),
             Flatten((InputDim(1), InputDim(2))),
         )
-        view_as_complex_rule = ops[torch.view_as_complex].dim_map(inp)
+        view_as_complex_rule = dim_maps[torch.view_as_complex](inp)
         self.assertEqual(view_as_complex_rule, expected_view_as_complex_rule)
         expected_view_as_real_rule = (
             InputDim(0),
             Split(InputDim(1), (13, 2), 0),
             Split(InputDim(1), (13, 2), 1),
         )
-        view_as_real_rule = ops[torch.view_as_real].dim_map(intermediate)
+        view_as_real_rule = dim_maps[torch.view_as_real](intermediate)
```

Comment:
I think we'll need to improve some of the existing test cases in this file to work with CommDebugMode, to make sure the communication that are happening is expected. This can be done in a follow up PR